lib/utils/score.dart

```diff
@@ -31,8 +32,27 @@ class Score {
     return uuid;
   }
 
+  static incrementGamesCounter() async {
```

Comment:
games should also be counted on mobile.

Reply:
@abegehr what makes you think they are not?

Reply:
this is merged into the coronajump branch only